### DISCUSSION TITLE
Custom Domain

### DISCUSSION THREAD

**Discussion by arnoldsigik** [thedevs-network/kutt#503] [General]
Hi, any reply is welcomed.

I tried to add a custom domain to my kutt.anydomain.name, just say mydomain.name. 
Yes, I followed the instruction on kutt.anydomain.name/settings, pointing A and www of mydomain.name to 192.64.116.170. Then added the mydomain.name in the page of kutt.anydomain.name/settings

But I found out that the mydomain.name got redirected to kutt.it instead of kutt.anydomain.name.
Also, I notice that I could actually create the short url using mydomain.name, but it is pointing to https://kutt.it/404

Any idea what's wrong with this?

**Comment by poeti8** [comment]
Is `kutt.anydomain.name` your own hosted version of Kutt? If it is, then you should point your domain to your server IP instead of what is stated on the settings page, because that one is for the `kutt.it`'s server.
